### PR TITLE
docs(android): fix permission example for Android 8

### PR DESCRIPTION
**Pull request by m1ga**
**Optional Description:**

Add `WRITE_EXTERNAL_STORAGE` to the `requestPermissions` permission example so it will work on Android 8+ phones

### CODE REVIEW

File: apidoc/Titanium/Android/Android.yml [hunk 1]
```diff
@@ -108,10 +108,10 @@ description: |
     Starting from Android 6.0 (API level 23), users need to grant certain permissions to apps while the
     app is running. You can read it more [here](https://developer.android.com/training/permissions/requesting.html).
 
-    In Titanium SDK, to support this, we have the <Titanium.Android.requestPermissions> method. It is used to 
+    In Titanium SDK, to support this, we have the <Titanium.Android.requestPermissions> method. It is used to
     request any permission you may need. An example of using it is shown below:
 
-        var permissions = [ 'android.permission.CAMERA', 'android.permission.READ_EXTERNAL_STORAGE' ];
+        var permissions = [ 'android.permission.CAMERA', 'android.permission.READ_EXTERNAL_STORAGE', 'android.permission.READ_EXTERNAL_STORAGE' ];
```
Comment (drauggres) [inline apidoc/Titanium/Android/Android.yml]: `WRITE_EXTERNAL_STORAGE`

**Review by drauggres — CHANGES_REQUESTED** [review]

File: apidoc/Titanium/Android/Android.yml (hunk 1 above)
Comment (m1ga) [reply apidoc/Titanium/Android/Android.yml]: one small change and still I made a Copy&paste error 🙈 Thanks

**Review by garymathews — APPROVED** [review]
LGTM!

**Comment by sgtcoolguy** [comment]
BTW, we have integrated doc changes on master to an automated doc site deployment - so this change is already live at https://docs.appcelerator.com/platform/latest/#!/api/Titanium.Android (usually < 30 minutes of merge, but docker hub was done mid-day and messed it up)

**Comment by m1ga** [comment]
@sgtcoolguy nice, sounds good

**Pull request merged**